### PR TITLE
Updated contribution.md

### PR DESCRIPTION
**Pull request by Rjchauhan18**
#### Reason for change

As metioned In Issue https://github.com/Arelle/Arelle/issues/837 I have fixed the fork and pull request  long version of  descriptions to tldr version . 

#### Description of change

- I have change thte [contribution.md](https://github.com/Arelle/Arelle/blob/master/CONTRIBUTING.md) file. 
- I have change the setup environment section and add the drop downs and shorten everything.

@austinmatherne-wk Please review it and tell me if any changes needed.

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,23 +35,87 @@ You'll be working with Python. The Arelle implementation aims to support all sta
...
+
+<details>
+
+  <summary> Clone your fork </summary>
+<br>
+
+
+  ```
+  git clone https://github.com/`<your-github-username>`/Arelle.git
+
```
Comment (austinmatherne-wk) [inline CONTRIBUTING.md:56]: nitpick: the empty line here adds extra spacing for this snippet compared to the others.

**Review by austinmatherne-wk — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (Rjchauhan18) [reply CONTRIBUTING.md]: Hey @austinmatherne-wk ,
I have removed extra space in code snippet of git clone  url.
Can you please review it  again ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -35,23 +35,86 @@ You'll be working with Python. The Arelle implementation aims to support all sta
...
+</details>
+
+<details>
+
+  <summary> Clone your fork </summary>
+<br>
+
+
+  ```
+  git clone https://github.com/`<your-github-username>`/Arelle.git
```
Comment (brettkail-wk) [inline CONTRIBUTING.md:55]: The backquotes are unnecessary/wrong inside the code block:
image

**Comment by Rjchauhan18** [comment]
Hello @brettkail-wk 
I have removed unwanted backquotes.
Please review it again?

**Comment by Rjchauhan18** [comment]
#  **Suggestion** 

Contributor section in  readme file.


## 👥 Contributors

<div align="center">
  <a href="https://github.com/Arelle/Arelle/graphs/contributors">
    
  </a>
  <p>Join our community and become a contributor today! 🚀 </p>
</div>

---

And I also want to do one request that if possible can you please add the Hactoberfest Topic in repositery and hactoberfest label this PR .

**Comment by austinmatherne-wk** [comment]
The contributors suggestion is welcome if you'd like to add it. Otherwise I'll make sure this is merged soon. Thanks!

**Comment by Rjchauhan18** [comment]
Hello @austinmatherne-wk 
I have added contributer section in readme.md file .
Can you please review it ?

**Review by austinmatherne-wk — APPROVED** [review]

**Pull request merged**